charts/argus/templates/_helpers.tpl

```diff
@@ -130,4 +130,67 @@ Collector Pod security context
 capabilities:
   drop: {{ toYaml .Values.collector.securityContext.capabilities.drop | nindent 4 }}
   add: {{ toYaml $addCaps | nindent 4 }}
-{{- end }}
\ No newline at end of file
+{{- end }}
+
+{{/*
+LM Credentials and Proxy Details
+*/}}
+
+{{- define "lm-credentials-and-proxy-details" -}}
+{{- $secretObj := (lookup "v1" "Secret" .Release.Namespace .Values.global.userDefinedSecret) | default dict }}
+{{- $secretData := (get $secretObj "data") | default dict }}
```

Comment:
where are we using this retrieved object?
Good to add some comments here mentioning the code flow.

Reply:
Noted